src/Lecture2.hs

```diff
@@ -181,7 +298,11 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True 
+isIncreasing (x:y:xs)
+  | x < y = isIncreasing (y:xs)
+  | otherwise = False
+isIncreasing _ = True 
```

Comment:
You can the implementation slightly more elegant by relying on laziness in Haskell 🦥 

```suggestion
isIncreasing [_] = True
isIncreasing (x : y : ys) = x < y && isIncreasing (y : ys)
```